lib/src/charts/data/bars/bar_interactions.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:chart_it/src/charts/data/bars/bar_data.dart';
+import 'package:chart_it/src/charts/data/bars/bar_group.dart';
+import 'package:chart_it/src/interactions/interactions.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter/gestures.dart';
+
+class BarInteractionEvents
+    extends TouchInteractionEvents<BarInteractionResult> {
+  final bool snapToNearestBar;
+  final double fuzziness;
+
+  const BarInteractionEvents({
+    required super.isEnabled,
+    this.snapToNearestBar = true,
+    this.fuzziness = 0.0,
+    super.onTap,
+    super.onDoubleTap,
+    super.onDragStart,
+    super.onDrag,
+    super.onDragEnd,
+  }) : assert(
+          fuzziness > 0.0 ? snapToNearestBar : true,
+          'For fuzziness to work, snapToNearestPoint should be enabled!',
+        );
```

Comment:
This is not true anymore. Let's remove the assert

Reply:
Good catch. Removed